tests/test_node.py

```diff
@@ -1,16 +1,18 @@
+from os import symlink
+
 import pytest
 
 from scantree import CyclicLinkedDir, DirNode, LinkedDir, RecursionPath
 from scantree.test_utils import get_mock_recursion_path
 
 
-def create_basic_entries(local_path):
+def create_basic_entries(local_path: str):
```

Comment:
This is not a `str`, but a `py.path.local` gotten via the `tmpdir` fixture https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmpdir-fixture. The type hint here is misleading since `local_path.join` would do something very different if `local_path` is a string :)

(We should actually use the newer [tmp_path](https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmp-path-fixture) fixture everytwhere, but in a separate PR!)

Reply:
reverted